config/processCcd.py

```diff
@@ -56,12 +56,15 @@
 config.charImage.load(os.path.join(configDir, "cmodel.py"))
 config.charImage.measurement.load(os.path.join(configDir, "apertures.py"))
 config.charImage.measurement.load(os.path.join(configDir, "kron.py"))
+config.charImage.measurement.load(os.path.join(configDir, "hsm.py"))
+if "ext_shapeHSM_HsmShapeRegauss" in config.charImage.measurement.plugins:
+    # no deblending has been done
+    config.charImage.measurement.plugins["ext_shapeHSM_HsmShapeRegauss"].deblendNChild = ""
+    config.charImage.measurement.slots.shape = "base_SdssShape"
```

Comment:
Actually, looking at your **/tigress/HSC/HSC/rerun/hsc-1382/20151209/cosmos/config/processExposure.py**, HSM measurements are not in the list of algorithms: `processCcd.calibrate.initialMeasurement.algorithms.names=['flux.psf', 'flags.pixel', 'flux.naive', 'flux.gaussian', 'centroid.naive', 'flux.sinc', 'shape.sdss', 'skycoord', 'classification.extendedness']`, so it's not actually being run at this stage.

So, yes, this slot should be changed to `ext_shapeHSM_HsmSourceMoments`.  Now, the question is: do we really want to be running HSM at charImage stage, or just the calibrate stage?  I'm still a bit confused about the mapping from initialMeasurement/calibrate/mesurement (HSC) --> charImage/calibrate (LSST).  It's the initialMeasurement stage on HSC that does not run HSM (the other two do and both have the shape slot of `shape.hsm.moments`).